src/Libraries/RevitNodes/Elements/AdaptiveComponent.cs

```diff
@@ -127,8 +316,7 @@ public List<Point> Locations
         // without breaking replication guides
 
         [IsVisibleInDynamoLibrary(false)]
-        [AllowRankReduction]
-        public static AdaptiveComponent[] ByParametersOnFace(double[][][] uvs, ElementFaceReference faceReference, FamilyType familyType)
+        public static AdaptiveComponent ByParametersOnFace(double[][] uvs, ElementFaceReference faceReference, FamilyType familyType)
```

Comment:
ByParametersOnFace has 4 overloaded functions. This is hidden originally. I guess it is because actually it is only supposed to be supplementory to the other one with the same number of parameters. So if the input is not Geometry.UV[], it still can work.